ssm/crypto.py

```diff
@@ -310,22 +319,22 @@ def get_certificate_subject(certstring):
 
 
 def get_signer_cert(signed_text):
-    '''
-    Read the signer's certificate from the signed specified message, and return the
-    certificate string.
-    '''
+    """Read the signer's certificate from the signed specified message."""
```

Comment:
Why lose the reference to what the method returns?

```suggestion
    """Returns the signer's certificate from the signed specified message."""
```

Reply:
I was just trying to remove redundant information, but yes, I should have kept the info on what it returns. It should be in the imperative mood though.

Reply:
I've updated the suggestion above (can't seem to add a suggestion in a comment under a comment)